src/diffpy/labpdfproc/tools.py

```diff
@@ -48,6 +48,11 @@ def expand_list_file(args):
             file_inputs = [input_name.strip() for input_name in f.readlines()]
         args.input.extend(file_inputs)
         args.input.remove(file_list_input)
+    wildcard_inputs = [input_name for input_name in args.input if "*" in input_name]
+    for wildcard_input in wildcard_inputs:
+        input_files = [str(file) for file in Path(".").glob(wildcard_input) if "file_list" not in file.name]
```

Comment:
I think we may be able to remove `if file_list not in file.name` because file_list files have been removed already.

Reply:
I think this is for files in the glob directory (not in args.input), so if we have a file list in the same directory as the wildcard, then it'll be loaded if we don't skip it.

Reply:
I see, yes better to have it to be on the safe side.   It may be better to make it stricter so `if file.name == file_list.txt` but it is probably ok as it is.